scripts/waf-ip-set-sync.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo "Generating IP set"
+
+IPV4_LIST=$(grep -v '^#' allowed_ips.txt | jq -Rs '{Addresses: split("\n") | map(select(length > 0))}')
+
+echo "Fetching IP set IDs"
+
+{
+  read -r IPV4_SET_ID
+  read -r IPV6_SET_ID
+} < <(aws wafv2 list-ip-sets --scope REGIONAL | jq '.IPSets[] | select( .Name | contains("api-customers")) | .Id')
+
+echo "Updating IPv4 set"
+
+LOCK_TOKEN=$(aws wafv2 get-ip-set --name $APP-$ENV-api-customers --scope REGIONAL --id $IPV4_SET_ID --region us-east-1 | jq '.LockToken')
+
+aws wafv2 update-ip-set \
+  --name $APP-$ENV-api-customers \
+  --scope REGIONAL \
+  --id $IPV4_SET_ID \
+  --region us-east-1 \
+  --addresses $IPV4_LIST \
+  --lock-token $LOCK_TOKEN
```

Comment:
Based on the note at https://docs.aws.amazon.com/waf/latest/APIReference/API_UpdateIPSet.html it appears that update-ip-set will replace all existing addresses in the set, which is what we want, but we should confirm that behavior.